frontend/src/context/SpotlightifyContext.tsx

```
@@ -31,73 +31,96 @@
-        return {
-          ...state,
-          commandHistory: [action.payload],
-          activeCommand: action.payload,
-        };
+      const pushedCommandHistory = [
+        ...state.commandStack,
+        {
+          command: action.payload.command,
+          options: action.payload.options || {},
+        },
+      ];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)